### PR TITLE
CE-33707: Fixed deprecated methods for PHPUnit v9.3.0

### PR DESCRIPTION
**Pull request by anzin**
### Description (*)
This PR fixes CE project codebase compatibility problems with the PHPUnit v.9.3.0.
See changes here: [PHPUnit 9.3.0 ChangeLog](https://github.com/sebastianbergmann/phpunit/blob/9.3.0/ChangeLog-9.3.md#configuration-of-code-coverage-and-logging-in-phpunitxml)

In this PR was changed 183 files where the main problem was using at() matcher that has been deprecated.

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/576
https://github.com/magento/partners-magento2b2b/pull/579

### Related Issues
1. magento/magento2#33707

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by anzin** [comment]
@magento run all tests

File: app/code/Magento/Cms/Test/Unit/Controller/Adminhtml/Wysiwyg/DirectiveTest.php [hunk 1]
```diff
@@ -106,6 +106,9 @@ class DirectiveTest extends TestCase
      */
     private $driverMock;
 
+    /**
+     * @inheirtDoc
```
Comment (eduard13) [inline app/code/Magento/Cms/Test/Unit/Controller/Adminhtml/Wysiwyg/DirectiveTest.php]: Typo here and in all the `setUp` methods below. Also, I don't think that we should camelcase it, based on the [phpdoc manual](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html).
```suggestion
     * @inheritdoc
```

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run Static Tests

**Comment by anzin** [comment]
@magento run Static Tests

**Comment by anzin** [comment]
@magento run Static Tests

**Comment by anzin** [comment]
@magento run Static Tests

**Comment by anzin** [comment]
@magento run all tests

**Review by andrewbess — APPROVED** [review]

**Pull request merged**